storybook/decorators/with-theme.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Internal dependencies
+ */
+import Theme from '../../packages/components/src/theme';
+
+/**
+ *  A Storybook decorator to show a div before and after the story to check for unwanted margins.
+ */
+
+export const WithTheme = ( Story, context ) => {
+	const themes = {
+		default: {},
+		modern: {
+			accent: '#3858e9',
+		},
+		sunrise: {
+			accent: '#dd823b',
+		},
```

Comment:
we may want to add a comment here justifying why we chose these two colors (like explained in the PR's description)